build-tools/scripts/RunTests.targets

```diff
@@ -83,5 +83,8 @@
         Properties="AotAssemblies=True"
     />
   </Target>
-  <Target Name="RunAllTests" DependsOnTargets="RunNUnitTests;RunJavaInteropTests;RunApkTests" />
+  <Target Name="RunPerformanceTests">
+    <MSBuild Projects="$(MSBuildThisFileDirectory)..\timing\timing.csproj" Targets="MSBuildTiming" />
+  </Target>
+  <Target Name="RunAllTests" DependsOnTargets="RunNUnitTests;RunJavaInteropTests;RunApkTests;RunPerformanceTests" />
```

Comment:
Let's move the `DependsOnTargets` values into a new `$(RunAllTestsDependsOn)` property, so that every time we add a new target we don't need to repeat quite so much.